### ISSUE TITLE
module.resource is undefined in Webpack

### ISSUE THREAD

**Issue by sporto** [opvasger/elm-minify#15]
Hi
I tried the webpack plugin in our project. But got this error:

```
/usr/src/app/node_modules/elm-minify/src/api.js:63
                    if (module.resource.endsWith(".elm") === false) return;
                                        ^

TypeError: Cannot read property 'endsWith' of undefined
    at /usr/src/app/node_modules/elm-minify/src/api.js:63:41
    at Array.forEach (<anonymous>)
    at /usr/src/app/node_modules/elm-minify/src/api.js:61:25
```

From https://github.com/opvasger/elm-minify/blob/3eda99632e7aa06b68a388044e168f0206410d31/src/api.js#L63

Thanks

**Comment by joecorkerton** [comment]
Just ran into the same error!

**Comment by opvasger** [comment]
This is my bad - sometimes I forget that I'm not compiling with Babel. Will patch this right now :sunny:

**Comment by opvasger** [comment]
this should now be patched with 1.5.2, which I've just released

**Comment by opvasger** [comment]
As per #16 I've changed how the plugin detects if it should minify a module, which is also the root cause of this issue 👍
